front-end/src/app/shared/components/transaction-type-base/transaction-memo.utils.ts

```diff
@@ -9,6 +9,7 @@ export class TransactionMemoUtils {
       if (text && text.length > 0) {
         const memo_text = MemoText.fromJSON({
           text4000: text,
+          text_prefix: transaction.memo_text?.text_prefix,
```

Comment:
Would it make sense to refactor the two static functions in in this file and put them in the additional-info-input component? The transaction and form are available in the component and then memo HTML and component code are self-contained. Then we could eliminate this file:

https://github.com/fecgov/fecfile-web-app/blob/develop/front-end/src/app/shared/components/transaction-type-base/transaction-memo.utils.ts

Reply:
They're not used in the additional info component, but I've moved them to the one place where they are used, the transaction form utils.